LekceDveSamples/LekceDveSamples/UserLoginTests.cs

```diff
@@ -0,0 +1,35 @@
+using FluentAssertions;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace LessonTwoSamples
+{
+    public class UserLoginTests
+    {
+        public WebDriver webDriver = new ChromeDriver();
+        string baseUrl = "";
+
+
+        [Test]
+        public void GivenLoginPageIsLoaded_WhenUserLoginWithValidCredentials_ThenIsLoggedIn()
+        {
+            webDriver.Navigate().GoToUrl(baseUrl);
+            webDriver.FindElement(By.Id("LoginField")).SendKeys("Ucastnice");
+            webDriver.FindElement(By.Id("PasswordField")).SendKeys("Heslo");
+            webDriver.FindElement(By.Id("OkButton")).Click();
+            webDriver.FindElement(By.Id("WelcomeBanner")).Text
+                .Should().Be("Prihlaseni problehlo uspesne");
+        }
+
+        [Test]
+        public void ValidUserCanLoginIn()
+        {
+            webDriver.Navigate().GoToUrl(baseUrl);
+            webDriver.FindElement(By.Id("LoginField")).SendKeys("Ucastnice");
+            webDriver.FindElement(By.Id("PasswordField")).SendKeys("Heslo");
+            webDriver.FindElement(By.Id("OkButton")).Click();
+            webDriver.FindElement(By.Id("WelcomeBanner")).Text
+                .Should().Be("Prihlaseni problehlo uspesne");
```

Comment:
jenom bych to vyextrahoval do konstanty ať se to mění na jedno místě

Reply:
Jako v normalnim projektu bych to tak udelal, ale tady v sample souboru ktery je jeste prvni sveho druhu bych to spise nechal takhle, pro lepsi citelnost